Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -207,6 +207,15 @@ function New-LDAPIdentitySource {
     if (($SecondaryUrl -match '^(ldap:).+((:636)|(:3269))$')) {
```

Comment:
Since we're validating it further down anyway, probably should remove this.

Reply:
should be good now.